app/lib/actions/getPharmacy.js

```diff
@@ -0,0 +1,18 @@
+const apiRequest = require('../etl-toolkit/apiRequest');
+const fixPhoneNumbers = require('../fixPhoneNumbers');
+const config = require('../config');
+
+function toObject(pharmaString, odsCode) {
+  const pharmacy = JSON.parse(pharmaString);
+  // eslint-disable-next-line no-underscore-dangle
+  pharmacy._id = odsCode;
```

Comment:
Fair points. I could see there was a need for an id that was under our control as opposed to just using the randomly named id of whatever data was being imported. I wanted to make sure `_id` was a conscious decision, seems like it is.